1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
+
+ENV GOLANG_VERSION 1.6.1
+ENV GOLANG_DOWNLOAD_URL "https://golang.org/dl/go$GOLANG_VERSION.windows-amd64.zip"
```

Comment:
It's structured the same was as the Linux Dockerfiles: https://github.com/docker-library/golang/blob/master/1.6/Dockerfile#L11